pkg/handlers/webhookreceiver.go

```diff
@@ -168,13 +170,14 @@ func (h *WebhookReceiverHandler) processAlert(alert template.Alert, mnl *oav1alp
 		if err != nil {
 			log.WithFields(log.Fields{LogFieldNotificationName: notification.Name, LogFieldManagedNotification: managedNotifications.Name}).WithError(err).Error("unable to update notification status")
 		}
-		metrics.SetResponseMetricFailure("service_logs")
+		// Set the metric for failed service log response from OCM
+		metrics.SetResponseMetricFailure(config.ServiceLogService, notification.Name)
```

Comment:
What do you think of including an alert name label here as well?

Reply:
Only problem is that, the metric is not only used by service logs. It is also used when the response failures of setting Limited support flag for the clusters. Hence I have only added one label. Let me know what you think.

Reply:
I have added an additional label as suggested.